packages/web/src/components/track/TrackTileMetrics.tsx

```diff
@@ -172,11 +173,14 @@ export const CommentMetric = (props: CommentMetricProps) => {
   const url = isMobile
     ? `${permalink}/comments`
     : `${permalink}?showComments=true`
+  const isSmall = size === TrackTileSize.SMALL
 
   return (
     <VanityMetric ellipses to={url} onClick={handleClick}>
       <IconMessage size='s' color='subdued' />
-      {comment_count > 0 ? formatCount(comment_count) : 'Leave a comment'}
```

Comment:
Nice, technically not the exact spec but I like this more anyway

Reply:
nah design asked for this

Reply:
Sg, I'll follow up with them on Monday just to confirm changing the Figma design then

Reply:
https://www.figma.com/design/gGVAGoOfp1hcpHCvmHKRUh/%F0%9F%92%AC-Commenting?node-id=2760-55145&node-type=frame&m=dev

Reply:
it's already reflected & there was convo in the ticket